src/NUnitFramework/framework/Api/DefaultTestAssemblyBuilder.cs

```diff
@@ -164,10 +169,12 @@ private TestSuite Build(Assembly assembly, string assemblyPath, IDictionary<stri
                     }
                 }
 
-                IList fixtureNames = null;
+                _filter = new PreFilter();
                 if (options.ContainsKey(FrameworkPackageSettings.LOAD))
-                    fixtureNames = options[FrameworkPackageSettings.LOAD] as IList;
-                var fixtures = GetFixtures(assembly, fixtureNames);
+                    foreach (string filterText in (IList<string>)options[FrameworkPackageSettings.LOAD])
```

Comment:
We're going from requiring `IList` to requiring `IList<string>`. E.g. ArrayList would have worked before but now would be ignored. Is there any chance of this being a problem?

Reply:
It's a problem if you consider this a public API. I don't, because it's something that can only be accidentally discovered by virtue of reading pretty deeply in the framework code. Bit if you want it to be treated as an API, that's possible and we should then use ILIst.

Reply:
I'm not the most familiar, so just confirming this so I can keep up: you don't expect the constant `FrameworkPackageSettings.LOAD` to be used by third parties integrating with NUnit in such a way that their values could end up here?

Reply:
I don't. However, in an earlier discussion on the issue itself, both you and @ChrisMaddock asked me to keep the existing `LOAD` setting to avoid even the small risk of breaking anything. I suppose keeping `IList` would be consistent with that approach.